ciphey/basemods/Searchers/ausearch.py

```diff
@@ -310,6 +314,7 @@ def __init__(self, config: Config):
         self.max_depth = int(self._params()["max_depth"])
         if self.max_depth == 0:
             self.max_depth = math.inf
+        self.p_threshold = float(self._params()["p_threshold"])
```

Comment:
Where does `self._params()["p_threshold"]` come from in the code base?

Reply:
It's a parameter to the searcher

Reply:
Found it!

```
            "p_threshold": ParamSpec(
                req=False,
                desc="Will skip any crackers which have less than this likelihood of succeeding. "
                     "Set to 0 to disable",
                default="0.01",
            ),
```

noice